tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature

```diff
@@ -26,6 +26,40 @@ Feature: Share by public link
     When the public uses the webUI to access the last public link created by user "user1"
     Then file "lorem.txt" should be listed on the webUI
 
+  @smokeTest
+  Scenario Outline: simple sharing by public link with different roles
+    When the user creates a new public link for folder "simple-folder" using the webUI with
+      | role | <role> |
+    Then user "user1" should have a share with these details:
+      | field       | value          |
+      | share_type  | public_link    |
+      | uid_owner   | user1          |
+      | permissions | <permissions>  |
+      | path        | /simple-folder |
+      | name        | Public link    |
+    And a link named "Public link" should be listed with role "<role>" in the public link list of folder "simple-folder" on the webUI
+    When the public uses the webUI to access the last public link created by user "user1"
+    Then file "lorem.txt" should be listed on the webUI
+    Examples:
+      | role        | permissions                  |
+      | Viewer      | read                         |
+      | Contributor | read, change, create, delete |
```

Comment:
as discussed in the other ticket, this is wrong ?
https://github.com/owncloud/phoenix/issues/1979

Reply:
fix is here https://github.com/owncloud/phoenix/pull/1985. please adjust expectation